### ISSUE TITLE
Support installation on vanilla edx/edx-platform and configuration

### ISSUE THREAD

**Issue by OmarIthawi** [appsembler/figures#3]
This would be a stretch goal, but I believe it can be done. This would seriously help wide adoption in the community.

**Comment by johnbaldwin** [comment]
@OmarIthawi You think we can retire this card?

**Comment by OmarIthawi** [comment]
Or course @johnbaldwin, once https://github.com/appsembler/figures/pull/84 is merged.

**Comment by johnbaldwin** [comment]
Issue fixed with PR #84
